runtime/java/src/common/FileRange.java

```diff
@@ -0,0 +1,87 @@
+//package agnosis-context.libs.visualization;
```

Comment:
This class doesn't seem to be used anywhere - remove?  Seems redundant with `Location` anyway.